tests/unit/model/test_model.py

```diff
@@ -293,6 +295,44 @@ def test_feature_columns(object_under_test):
     assert object_under_test.feature_columns == expected
 
 
+def test_from_api_repr_w_transform_columns(target_class, resource):
```

Comment:
Let's use a `Model` object called `object_under_test` in the tests above to test the `Model.transform_columns` property.